content/en/docs/Learning Resources/Blog Posts/_index.md

```diff
@@ -9,3 +9,5 @@ description: >
 * [Apache Parquet: Cost Efficiency @ Scale in Big Data File Format](https://eng.uber.com/cost-efficiency-big-data/)
 
 * [One Stone, Three Birds: Finer-Grained Encryption @ Apache Parquet](https://eng.uber.com/one-stone-three-birds-finer-grained-encryption-apache-parquet/)
+
+* [Designing a Parquet Catalog for InfluxDB IOx](https://www.influxdata.com/blog/designing-a-parquet-catalog-for-influxdb-iox/)
```

Comment:
I am not sure the publishing process is supposed to work, but this blog post does not appear to be on the parquet.apache.org site https://parquet.apache.org/docs/learning-resources/blog-posts/

🤔 

![Screenshot 2023-04-27 at 12 40 35 PM](https://user-images.githubusercontent.com/490673/234931563-e488cc4e-4b92-454f-bc50-16c3c9034ca0.png)

Reply:
Hmm... That's weird. I can see the link locally.

Reply:
> Hmm... That's weird. I can see the link locally.

As in you can see it when the docs are built locally or you can see it when you open https://parquet.apache.org/docs/learning-resources/blog-posts/ locally 🤔

Reply:
I can see it from the locally built site. I have no idea why it is missing from the published one.